_includes/manuals/1.7/4.3.1_smartproxy_installation.md

```diff
@@ -7,9 +7,65 @@ can be found on the [**Foreman Architecture**](manuals/{{page.version}}/index.ht
 To fully manage the commissioning process then a smart proxy will have to manipulate these services, DHCP, DNS, Puppet CA, Puppet and TFTP. These services may exist on separate machines or several of them may be hosted on the same machine. As each smart proxy instance is capable of managing all the of these services, there is only need for one proxy per host.
 In the special case of a smart proxy managing a windows DHCP server, the host machine must be running Windows and support the **netsh dhcp** utility, it does not need to be the Microsoft DHCP server itself.
 
-#### Packages
-
-RPM and Debian packages are available, see the [Install from Packages](manuals/{{page.version}}/index.html#3.3InstallFromPackages) section for configuration and install the `foreman-proxy` package.
+By default, our installer deploys one Foreman Proxy instance on the same host
+Foreman was installed on. This chapter describes installation of additional
+instance(s).
+
+#### Installer
```

Comment:
I haven't noticed we have that!

Dropped all my changes except the manual installation paragraph which is now in the 3.2.3.